test/features/qdevice_setup_remove.feature

```diff
@@ -32,6 +32,8 @@ Feature: corosync qdevice/qnetd setup/remove process
     Then    Cluster service is "started" on "hanode2"
     And     Online nodes are "hanode1 hanode2"
     And     Service "corosync-qdevice" is "stopped" on "hanode2"
+    When    Run "echo "# This is test for bsc#1166684" >> /etc/corosync/corosync.conf" on "hanode1"
```

Comment:
Update `This is test...` by `This is a test..`

Reply:
Changed and sync to other two branches